.github/actions/next-integration-stat/dist/37.index.js

```diff
@@ -0,0 +1,452 @@
+"use strict";
```

Comment:
Do we mean to check these in?

Reply:
This is ncc-compiled output after updating dependencies, so I expect it is needed. I don't know why upgrading some dep creates separate chunk though.